jmix-data/eclipselink/src/main/java/io/jmix/eclipselink/impl/JmixEclipselinkJtaTransactionManager.java

```diff
@@ -20,8 +20,9 @@
 import jakarta.transaction.TransactionManager;
 import jakarta.transaction.UserTransaction;
 import org.eclipse.persistence.internal.helper.JmixUtil;
+import org.springframework.beans.BeansException;
+import org.springframework.context.ApplicationContext;
 import org.springframework.transaction.TransactionDefinition;
-import org.springframework.transaction.support.DefaultTransactionStatus;
 
 public class JmixEclipselinkJtaTransactionManager extends JmixJtaTransactionManager {
```

Comment:
Unused

Reply:
This class is used by a global transactions mechanism. We have an article about this in YT KB.